frontend/src/js/controllers/challengeCtrl.js

```diff
@@ -984,6 +984,99 @@
         }
         };
 
+        var template = '<section class="ev-md-container text-center rm-overflow-y">'
```

Comment:
Do you really want to do it in this ugly manner? Cannot we have it in a view please?